util/gc.py

```diff
@@ -37,6 +37,10 @@ def get_free():
     stats = os.statvfs(ROOT_DIR)
     return stats.f_bavail * stats.f_bsize / pow(1024,3)
 
+  #The callback executed when shutil methods fail to run successfully
+  def onerror(func, path, exc_info):
+    print "ERROR: Cannot remove '%s' Check path permissions" % path 
```

Comment:
As a matter of best practice, this should probably be `% (path,)` to avoid unpleasant surprises when `path` is a tuple.